tests/README.md

```diff
@@ -0,0 +1,15 @@
+# Testing
+
+To run the unit test suite:
+
+```shell
+./setup.py test --test-suite tests
+```
+
+To run the integration tests:
+
+```shell
+dc up
+```
+
+(Note: The integration tests will display in the container output)
```

Comment:
I suspect we can do some more intuitive "stuff" here, but I figured this was at least helpful for the time being.